Automattic/wp-calypso
# Stats: Ship fixed nav headers to public

#### Proposed Changes

1. Enables the new fixed navigation headers across all environments.
2. Removes unused code paths related to old header style.

![Image](https://user-images.githubusercontent.com/390760/195898723-db3d2a33-1c13-4b6f-b285-24a9b628d6e6.png)

#### Testing Instructions



1. Checkout the Calypso live link.
2. Navigate to the Stats page for a given site.
3. Click on any of the clickable headers (like Pages & posts).
4. Confirm the new header style is in place.
5. From the Stats page, click an individual post under Pages & posts.
6. Confirm the new header is in place here too.

#### Pre-merge Checklist






Related to #69179
